versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx

```diff
@@ -446,7 +446,7 @@ Perform these steps:
    serviceexport.networking.kubeslice.io/reviews created
    ```
 :::info
-This completes the deployment of Bookinfo Appication on a Slice.**
+This completes the deployment of Bookinfo Application on a Slice.**
```

Comment:
Application can be `application` and Slice can be a `slice` as it is a paragraph (note).

Reply:
updated